polynote-runtime/src/main/scala/polynote/runtime/DataEncoder.scala

```diff
@@ -50,14 +51,43 @@ trait DataEncoder[@specialized T] extends Serializable {
 }
 
 object DataEncoder extends DataEncoder0 {
+scala.collection.immutable.HashMap().toString()
```

Comment:
What is this line for?

Reply:
Seems like cruft (probably I just wanted to check what `HashMap`'s `toString` method looked like, without digging through 200 traits to find where it was actually implemented).

Will remove